pipelines/managed/push-oot-kmods/push-oot-kmods.yaml

```diff
@@ -0,0 +1,454 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: push-oot-kmods
+  labels:
```

Comment:
Can you remove this? I'm removing them all here https://github.com/konflux-ci/release-service-catalog/pull/1097